layouts/shortcodes/cncf-landscape.html

```diff
@@ -57,7 +57,11 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
-  <iframe frameborder="0" id="landscape" scrolling="no" src="" style="width: 1px; min-width: 100%" title="CNCF Landscape"></iframe>
+  {{ if ( .Get "category" ) }}
+  <iframe frameborder="0" id="landscape" scrolling="no" src="https://landscape.cncf.io/category={{ .Get "category" }}&format=logo-mode&grouping=category&embed=yes"></iframe>
+  {{ else }}
+  <iframe frameborder="0" id="landscape" scrolling="no" src="https://landscape.cncf.io/format=logo-mode;embed=yes" style="width: 1px; min-width: 100%" title="CNCF Landscape"></iframe>
```

Comment:
The rendered page looks nice.
I am not familiar with the various values of `format`. What other options are there besides `logo-mode`?
Is there an example in the docs of a page that shows the entire landscape (else clause)?
What is the point of the `helpers` attribute?

Reply:
I added this updated shortcode to a content page. The landscape logos do not use up the page as expected (width).
I'd like the shortcode to be used without adding extra HTML to the Markdown page.